### PR TITLE
Add download setting for Kotlin to README

### PR DESCRIPTION
**Pull request by phicdy**
I confused the setting for kotlin, add the description for kotlin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +131,24 @@ dependencies {
 }
 ```
 
+For Kotlin
+
+```groovy
+apply plugin: 'kotlin-kapt'
+
+kapt {
+    generateStubs = true
```
Comment (hotchemi) [inline README.md]: Thx! I'd say this is kind of old-fashioned and no longer needed~!

**Comment by shiraji** [comment]
This is the second PR requesting kapt description in README.

See https://github.com/permissions-dispatcher/PermissionsDispatcher/pull/329#issuecomment-310020037

I'm not sure we should do this but if it helps, we can add that, now.

**Comment by phicdy** [comment]
oh, sorry for duplicated issue. It's ok to merge or close the issue.

**Comment by hotchemi** [comment]
Oh thx for pointing out!
I changed my mind a bit...now we have a dedicated doc for Kotlin support so that I'd say it's ok to add the setting stuff to this page. What do you think?

**Comment by shiraji** [comment]
Yea adding the description in that doc can be helpful. I like that idea better than adding this in README.

**Comment by phicdy** [comment]
OK, I'll move this change to the doc.

**Comment by phicdy** [comment]
Hi @hotchemi and @shiraji ,

I've updated the commit. Could you please check it ?

File: doc/kotlin_support.md [hunk 2]
```diff
@@ -4,12 +4,32 @@ From 3.0.0 we started support `.kt` file generation.
...
 
+Include the following in your **app module** `build.gradle` file:
+
+`${latest.version}` is [![Download](https://api.bintray.com/packages/hotchemi/maven/permissionsdispatcher/images/download.svg)](https://bintray.com/hotchemi/maven/permissionsdispatcher/_latestVersion)
+
+```groovy
+apply plugin: 'kotlin-kapt'
+
+kapt {
+    generateStubs = true
```
Comment (hotchemi) [inline doc/kotlin_support.md]: I guess this is no longer needed. This is for kapt2 but the latest ver is 3.
https://kotlinlang.org/docs/reference/kapt.html

**Comment by phicdy** [comment]
I've confirmed it works without `kapt { generateStubs = true }`, updated the doc. Please check it.

**Pull request merged**